### PR TITLE
Modified in the input scale from 0~255 to 0~1 in both VGG16 and VGG19 

### PR DESCRIPTION
**Pull request by OwenLiuzZ**
### Checklist
- [x] I've tested that my changes are compatible with the latest version of Tensorflow.
- [x] I've read the [Contribution Guidelines](https://github.com/tensorlayer/tensorlayer/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description

1.modified the input scale from 0~255 to 0~1 in both VGG16 and VGG19 .
2.delete the conv_layers function in both vgg16 and vgg19

### CODE REVIEW

**Comment by DEKHTIARJonathan** [comment]
In principle, I agree with the direction of this PR.
Be careful, as the network is changed, it needs to be retrained and the current weights won't be working

@zsdonghao

**Comment by OwenLiuzZ** [comment]
@DEKHTIARJonathan @zsdonghao  Yes, currently, most of the VGG models are using input nets with pixels values in the range of [0-1]  to avoid the network crashing. However，we just modified the input image value range rather than the input nets. So, @zsdonghao advised me to release VGG16V2 and VGG19V2 to get rid of this problem

**Comment by DEKHTIARJonathan** [comment]
Personally, I would agree with the idea of replacing old VGG network using range [0, 255] completely.
Releasing a v2 is confusing, if we have done something bad, it is better to remove it ;)

However, you can still implement a _sanity check_, in case people use data within [0, 255] you can raise a warning. @zsdonghao

**Comment by zsdonghao** [comment]
agree

File: example/tutorial_vgg16.py [hunk 1]
```diff
@@ -174,6 +140,10 @@ def fc_layers(net):
 
 img1 = imread('data/laska.png', mode='RGB')  # test data in github
 img1 = imresize(img1, (224, 224))
+# rescale pixels values in the range of 0-1
+img1 = img1/255.0
+if ((0 <= img1).all() and (img1 <= 1.0).all()) is False:
```
Comment (zsdonghao) [inline example/tutorial_vgg16.py]: it seems you did not x255 inside the vgg model.

Comment (OwenLiuzZ) [reply example/tutorial_vgg16.py]: Yes, in this demo, we just deal with the range of input image. It seem as the tutorial vgg19. Actually, the input value will also be [0,255]

**Comment by OwenLiuzZ** [comment]
@DEKHTIARJonathan ok, your idea is great. I will implement it from your advice :)

File: tensorlayer/models/vgg19.py [hunk 2]
```diff
@@ -50,182 +48,15 @@ class VGG19Base(object):
     """The VGG19 model."""
 
     @staticmethod
-    def conv_layers(net_in):
+    def vgg19_simple_api(net_in, end_with):
         with tf.name_scope('preprocess'):
             # Notice that we include a preprocessing layer that takes the RGB image
             # with pixels values in the range of 0-255 and subtracts the mean image
             # values (calculated over the entire ImageNet training set).
-            mean = tf.constant([103.939, 116.779, 123.68], dtype=tf.float32, shape=[1, 1, 1, 3], name='img_mean')
```
Comment (zsdonghao) [inline tensorlayer/models/vgg19.py]: 乘以 255 之后，减去mean 还是需要的吧

**Comment by DEKHTIARJonathan** [comment]
@zsdonghao same remark here about approving a PR not ready to be merged.

**Pull request merged**